service/frontend/service.go

```diff
@@ -414,6 +416,15 @@ func (s *Service) Start() {
 				s.logger.Fatal("Failed to serve HTTP API server", tag.Error(err))
 			}
 		}()
+	} else if s.config.EnableNexusAPIs() {
+		var action string
+		if os.Args[0] == "temporal" {
+			action = "To enable Nexus, start the server with: `temporal server start-dev --http-port 7243 --dynamic-config-value system.enableNexus=true`."
+		} else {
+			action = "To enable Nexus, follow these instructions: https://github.com/temporalio/temporal/blob/main/docs/architecture/nexus.md#enabling-nexus."
+		}
+
+		s.logger.Warn(fmt.Sprintf("system.enableNexus dynamic config is enabled but the HTTP API port has not been set. Starting with Nexus disabled. %s", action))
```

Comment:
nit: Can we not automatically start nexus with default `--http-port 7243` and show this error message only when that port is already occupied?

Reply:
No, this feature is still in pre-release and we were specifically requested not to enable it by default unfortunately.

Reply:
I meant if `s.config.EnableNexusAPIs()` is true (i.e nexus enabled) then assume a default port for it.

Reply:
Unfortunately, we were asked not to do this and ask users to explicitly enable HTTP until the HTTP API stabilizes.